### PR TITLE
DOC-10742 new archives

### PR DESCRIPTION
**Pull request by RichardSmedley**

### CODE REVIEW

**Comment by RichardSmedley** [comment]
Adding @simon-dew to check Swagger.

File: home/modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -93,6 +98,7 @@ endif::[]
 
 [.tile]
 .Node.js SDK
+* xref:4.1@nodejs-sdk:hello-world:start-using-sdk.adoc[Version 4.2]
```
Comment (osfameron) [inline home/modules/ROOT/pages/index.adoc:101]: 4.1 vs 4.2

**Review by osfameron — APPROVED** [review]
Looks good 👍

**Review by simon-dew — APPROVED** [review]
LGTM 👍  as far as cb-swagger. I cheekily changed the PR title to enable linking back to Jira.

**Pull request merged**